deploy/kubernetes/common-snapshot-controller.yaml

```diff
@@ -0,0 +1,26 @@
+# This YAML file shows how to deploy the common snapshot controller
```

Comment:
Can you split up this folder into 2? One for controller yamls, and one for sidecar yamls.

Also prefer removing "common" from the name everywhere.

Reply:
sure.